package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-jhipster-entity-audit",
-  "version": "3.3.0",
+  "version": "3.3.1",
```

Comment:
This will be done automatically during the release. Thus could you revert this change? Thanks :)

Reply:
Sorry for the late response, I didn't receive notification. How do you want to me to do that (revert only this changes) ? To be honest, it's my first pull request :)

Reply:
You can make a new commit that changes the version number back, and ideally squash it with the previous commit.